plans/monitoring_multiple.pp

```diff
@@ -0,0 +1,60 @@
+# @summary Disable monitoring for targets in multiple services
+#
+# @param [TargetSpec] targets
+#   Set of targets to run against.
+#
+# @param [Enum['enable', 'disable']] action
+#   What action to perform on the monitored targets:
+#
+#     - `enable` Resumes monitoring alerts
+#     - 'disable' Supresses monitoring alerts
+#
+# @param [Boolean] noop
+#   Flag to enable noop mode.
+#
+# @example Remote target definition for $monitoring_target
+#   vars:
+#     patching_monitoring_plan:
+#       - 'patching::monitoring_solarwinds'
+#       - 'patching::monitoring_prometheus'
+#
+#   groups:
+#     - name: solarwinds
+#       config:
+#         transport: remote
+#         remote:
+#           port: 17778
+#           username: 'domain\svc_bolt_sw'
+#           password:
+#             _plugin: pkcs7
+#             encrypted_value: >
+#               ENC[PKCS7,xxx]
+#       targets:
+#         - solarwinds.domain.tld
+#
+#     - name: prometheus
+#       config:
+#         transport: remote
+#         remote:
+#           username: 'domain\prom_user'
+#           password:
+#             _plugin: pkcs7
+#             encrypted_value: >
+#               ENC[PKCS7,xxx]
+#       targets:
+#         - prometheus.domain.tld
+#
+plan patching::monitoring_multiple (
+  TargetSpec                $targets,
+  Enum['enable', 'disable'] $action,
+  Array                     $monitoring_plan = get_targets($targets)[0].vars['patching_monitoring_plan'],
```

Comment:
you're going to need to use a different key here

Reply:
if you use `patching_monitoring_plan` as an array, it's going to fail when you call `bolt plan run patching`.

Instead this key should be something like `patching_monitoring_plan_multiple` or similar.

Your vars would then look like:
```
vars:
  patching_monitoring_plan: 'patching::monitoring_multiple'
  patching_monitoring_plan_multiple:
    - 'patching::monitoring_solarwinds'
    - 'patching::monitoring_prometheus'
 ```